setup.py

```diff
@@ -52,9 +52,9 @@ def parse_requirements(fname='requirements/runtime.txt', with_version=True):
     CommandLine:
         python -c "import setup; print(setup.parse_requirements())"
     """
+    import re
```

Comment:
No need to import `re` once again if it has been imported in the 3rd line of setup.py.